Cargo.lock

```diff
@@ -70,9 +70,9 @@ dependencies = [
 
 [[package]]
 name = "alloy-core"
-version = "0.8.15"
+version = "0.8.20"
```

Comment:
Did you mean to run `cargo update`? I'd avoid bumping deps in this PR if there is no strong reason for it